runtime/doc/options.txt

```diff
@@ -1392,6 +1392,22 @@ A jump table for the options with a short description can be found at |Q_op|.
 	based expansion (e.g., dictionary |i_CTRL-X_CTRL-K|, included patterns
 	|i_CTRL-X_CTRL-I|, tags |i_CTRL-X_CTRL-]| and normal expansions).
 
+						*'completefilterfunc'* *'cff'*
+'completefilterfunc' 'cff' string (default: empty)
+	This option specifies a function to be used for filtering insert
+	mode completion entries. The function must have the signature
+
+		function(prefix, entry),
+
+	where 'entry' is a completion candidate, 'prefix' is the text
```

Comment:
single-quotes in vim :help docs have a special meaning, they always refer to editor options.
```suggestion
	where "entry" is a completion candidate, "prefix" is the text
```